phpBB/config/default/routing/storage.yml

```diff
@@ -0,0 +1,22 @@
+phpbb_storage_avatar_controller:
```

Comment:
alright, personally I prefer https://github.com/phpbb/phpbb/blob/master/phpBB/config/default/routing/feed.yml but do what you prefer